05-interactive-dst.md

```diff
@@ -181,3 +181,23 @@ print_decay.printTree(cands[0])
 With our candidates in hand, it would be nice to be able to retrieve and 
 compute the variables we need for an analysis. On to [LoKi 
 functors](06-loki-functors.html)!
+
+> ## Fast DST browsing {.callout}
+>
+> While here we have discussed for pedagogical reasons all the configuration
+> options needed in order to browse a `DST` file, in your daily life as a
+> physicist is often useful to use the `bender` application that belongs to
+> the `Bender` project.
+>
+> For example, to explore the `DST` we could have simply done:
+>
+> ```
+> lb-run Bender v27r0p1 bender 00035742_00000001_1.allstreams.dst
+> ```
```

Comment:
Is there a need to specify a Bender version here? This step probably doesn't need to be ‘reproducible’ like the DV stuff.